### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by Kacper-W-Kozdon** [Kacper-W-Kozdon/slab#1]
<div class='markdown-heading'><h1 class='heading-element'>SLAB Project</h1><a id='user-content-slab-project' class='anchor' aria-label='Permalink: SLAB Project' href='#slab-project'><span aria-hidden='true' class='octicon octicon-link'></span></a></div><div class='markdown-heading'><h2 class='heading-element'>TODO list:</h2><a id='user-content-todo-list' class='anchor' aria-label='Permalink: TODO list:' href='#todo-list'><span aria-hidden='true' class='octicon octicon-link'></span></a></div><ul><li>[ ] Reenable plots in test_torch.</li></ul><ul><li>[ ] Resolve mismatch of the outputs.</li></ul><ul><li>[ ] Clean up type hints in methods in Contini.</li></ul><ul><li>[ ] Add a webhook to automatically update the list of issues on GitHub.</li></ul><ul><li>[ ] Source of the nans is in mode == 'sum' of G_func()- ensure the addition is done correctly.</li></ul><ul><li>[ ] Fix the source of nan in the argument of G_func().</li></ul><ul><li>[ ] Try to move more methods and attributes into BaseClass and put it in a separate module.</li></ul><ul><li>[ ] Add training.py to torch_modules/other.</li></ul><ul><li>[ ] Make dictionary with functions or a function factory calling training loop functions.</li></ul><ul><li>[ ] Clean up type hints in methods in tContini.</li></ul><ul><li>[x] nan comes from the ** (3/4) of a negative value. Add a check or a filter for negative.</li></ul>
